File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string s;
        for(int i=0;i<max(word1.size(),word2.size());i++){
            if(i<word1.size()&& i<word2.size()){
                s+=word1[i];
                s+=word2[i];  
            }else if(i< word1.size()){
               s+= word1[i];
            }else if(i<word2.size()){
                s+=word2[i];
            }
         
        }
       
        
        return s;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merged = ""
        i = j = 0
        while i < len(word1) and j < len(word2):
            merged += word1[i] + word2[j]
            i += 1
            j += 1
        merged += word1[i:] + word2[j:]
        return merged